.yarn/plugins/@yarnpkg/plugin-conditions.cjs

```diff
@@ -0,0 +1,8 @@
+/* eslint-disable */
```

Comment:
Might want to add the following snippet to `.gitattributes` to prevent github from tracking this as source code

```
/.yarn/** linguist-vendored
```

Reply:
Thanks, I had no idea about this